### PR TITLE
DEVEX-2345 Support for compute and egress limits during project creation

### PR DESCRIPTION
**Pull request by wormsik**

### CODE REVIEW

File: src/python/dxpy/scripts/dx.py [hunk 1]
```diff
@@ -5543,6 +5547,8 @@ def __call__(self, parser, namespace, values, option_string=None):
                                 action='store_true')
 parser_new_project.add_argument('--database-ui-view-only', help='Viewers on the project cannot access database data directly', default=False,
                                 action='store_true')
+parser_new_project.add_argument('--monthly-compute-limit', type=positive_integer, help='Monthly compute limit for the project')
```
Comment (XingGemini) [inline src/python/dxpy/scripts/dx.py:5550]: Monthly project spending limit for compute

File: src/python/dxpy/scripts/dx.py [hunk 2]
```diff
@@ -5543,6 +5547,8 @@ def __call__(self, parser, namespace, values, option_string=None):
...
+parser_new_project.add_argument('--monthly-egress-bytes-limit', type=positive_integer, help='Monthly limit of egress bytes for the project')
```
Comment (XingGemini) [inline src/python/dxpy/scripts/dx.py:5551]: Monthly project spending limit for egress (in Bytes)

**Review by XingGemini — CHANGES_REQUESTED** [review]
Updated the help text.

File: src/python/dxpy/scripts/dx.py (hunk 2 above)
Comment (wormsik) [reply src/python/dxpy/scripts/dx.py]: done

File: src/python/dxpy/scripts/dx.py (hunk 1 above)
Comment (wormsik) [reply src/python/dxpy/scripts/dx.py]: done

**Review by kpjensen — APPROVED** [review]

**Pull request merged**